kuma/javascript/src/payments/terms.jsx

```diff
@@ -0,0 +1,150 @@
+// @flow
+import * as React from 'react';
+
+import A11yNav from '../a11y/a11y-nav.jsx';
+import Header from '../header/header.jsx';
+import SubHeader from './subheaders/index.jsx';
+import Footer from '../footer.jsx';
+import Route from '../route.js';
+
+type PaymentTermsParams = {
+    locale: string,
+};
+export const title = 'MDN Web Docs payment terms';
+export default function PaymentsTermsPage() {
+    return (
+        <>
+            <A11yNav />
```

Comment:
We have 4 pages now that does this? I think you should take this time to invest in some tech debt pay-down and fix this. 
Make new container component that extracts this technique. So *this* component should become as simple as:

```javascript
export default function PaymentsTermsPage() {
   return <Page>
    ...Stuff exclusive to Payment terms...
   </Page>
}
```

Reply:
Thanks @peterbe for your suggestion-- I updated this PR with the changes.